python/pyarrow/tests/test_parquet.py

```diff
@@ -73,6 +73,10 @@ def _read_table(*args, **kwargs):
     return pq.read_table(*args, **kwargs)
 
 
+def assert_tables_equal(left, right):
```

Comment:
Do we need a `pyarrow.testing` module where we would put assorted utilities like this?

Reply:
We may, however this is a detail of this module because the roundtrip to Parquet now introduces extra Field metadata so that `result.equals(expected)` will fail unless the right metadata is set in `expected`